### ISSUE TITLE
vkCmdBlitImage Crash iOS

### ISSUE THREAD

**Issue by DiegoAce** [KhronosGroup/MoltenVK#341]
After submitting a command buffer involving vkCmdBlitImage if the extent of the source image (possibly dst too) is small then the program crashes with the following assertion:
```
validateNewTexture:89: failed assertion `BytesPerRow of a buffer-backed texture with pixelFormat(MTLPixelFormatBGRA8Unorm) must be aligned to 64 bytes, found bytesPerRow(144)'
```
This only crashes on iOS and not macOS. I edited copy_blit_image.cpp to produce the crash when running it from the API-Samples demo.

[copy_blit_image.cpp.zip](https://github.com/KhronosGroup/MoltenVK/files/2558627/copy_blit_image.cpp.zip)

**Comment by billhollings** [comment]
@DiegoAce This is fixed in PR #375. Please retest and close if working.

**Comment by DiegoAce** [comment]
Looks good! I didn't have any crashes. Thanks!
